core/envelope-sim/src/main/kotlin/fr/sncf/osrd/envelope_sim/etcs/ETCSBrakingSimulator.kt

```diff
@@ -36,15 +42,29 @@ interface ETCSBrakingSimulator {
 
     /** Compute the ETCS braking curves for each LoA, ordered by LoA offset. */
     fun computeSlowdownBrakingCurves(
-        mrsp: Envelope,
+        envelope: Envelope,
         limitsOfAuthority: Collection<LimitOfAuthority>
     ): LOABrakingCurves
 
     /** Compute the ETCS braking curves for each EoA, ordered by EoA offset. */
     fun computeStopBrakingCurves(
-        mrsp: Envelope,
+        envelope: Envelope,
         endsOfAuthority: Collection<EndOfAuthority>
     ): EOABrakingCurves
+
+    /**
+     * Compute the ETCS LoAs, for every slowdown on an ETCS portion of the path present in the
+     * envelope.
+     */
+    fun computeSlowdownLocations(envelope: Envelope): List<LimitOfAuthority>
```

Comment:
Here we want the initial MRSP, no?
```suggestion
    fun computeSlowdownLocations(mrsp: Envelope): List<LimitOfAuthority>
```

Reply:
Actually, every usage of it will indeed be the MRSP. And it wouldn't work for a normal envelope since the slowdown location would be marked as the first point of the already existing slowdown curve. Works for me!